lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -51,7 +51,9 @@ define((require, exports, module) ->
   AUTO_INSERT = { text: "$$", selection: [1, 1] }
   SKIP = { text: "", selection: [1, 1] }
   dollarsInsertionAction = (state, action, editor, session, text) ->
-    if text == '$' and not editor.inMultiSelectMode
+    if text == '$'
+      if editor.inMultiSelectMode
```

Comment:
why don't you make it the first statement in this function? Just like in deletion action